models/ticket_history/int_zendesk__field_history_pivot.sql

```diff
@@ -11,7 +11,7 @@
 }}
 
 {% if execute and flags.WHICH in ('run', 'build') -%}
-    {% set results = run_query('select distinct field_name from ' ~ var('field_history') ) %}
+    {% set results = run_query('select distinct field_name from ' ~ ref('stg_zendesk__ticket_field_history') ) %}
```

Comment:
Before the source package consolidation, `var('field_history')` pointed to the staging model. Now, it points to the source, so when you run this model with `zendesk_sources` populated, it fails on 'no source found' due to this [line](https://github.com/fivetran/dbt_zendesk/blob/main/models/staging/src_zendesk.yml#L11)